app/Http/Controllers/BookController.php

```diff
@@ -24,8 +25,12 @@ public function getBookById(int $id) {
                 'success' => false
             ], 404);
         }
+
+        $book->genre;
+        $book->reviews;
+
         return response()->json([
```

Comment:
error response needed if book id not found:
Error Response:

Code: 404 NOT FOUND
Content: {"message": "Book with id 999 not found"}

Reply:
It's above on lines 23-26?